src/perception/aruco_detect.hpp

```diff
@@ -25,29 +26,13 @@
 constexpr char const* ODOM_FRAME = "odom";
 constexpr char const* ROVER_FRAME = "base_link";
 
-/**
- * @brief Fiducials that are currently visible by the camera.
- */
-struct ImmediateFiducial {
-    int id = -1;
-    cv::Point2f imageCenter{};
-    std::optional<SE3> fidInCam;
-};
+using PointCloudPtr = pcl::PointCloud<pcl::PointXYZRGBNormal>::Ptr;
```

Comment:
`pcl::PointCloud<T>::Ptr` is already a shared_ptr alias, if you really insist on doing this i'd rather not have two levels of indirection and just explicitly set PointCloudPtr to shared_ptr<pcl::PointCloud<T>>